internal/logging/http.go

```diff
@@ -0,0 +1,63 @@
+package logging
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+
+	"github.com/rs/zerolog"
+)
+
+var strInvalidHeaderFieldValue = []byte("invalid header field value")
+
+type FilteredHTTPLogger struct {
+	zerolog.Logger
+}
+
+func (l FilteredHTTPLogger) Write(b []byte) (int, error) {
```

Comment:
What does this filter? I feel like we could use some tests around this, it seems high risk for logging things that should not be logged.

Reply:
It filters out unknown header values from logs for https://github.com/infrahq/infra/issues/655. The code was moved from `logging/logger.go` and the test lives in `logging/logger_test.go`. I guess I can move the test to `logging/http.go`